cmd/gke-exec-auth-plugin/main.go

```diff
@@ -33,6 +35,8 @@ var (
 	altTokenURL  = flag.String("alt-token-url", "", "URL to token endpoint.")
 	altTokenBody = flag.String("alt-token-body", "", "Body of token request.")
 
+	flockPath = flag.String("flock-path", "/tmp/gke-exec-auth-plugin.lock", "Path to filesystem lock file.")
```

Comment:
Use `os.TempDir` and `filepath.Join` to construct default value

Reply:
Done.